editor/editor_help_search.cpp

```diff
@@ -401,8 +401,46 @@ bool EditorHelpSearch::Runner::_phase_match_classes() {
 					}
 				}
 			}
-			matches[class_doc.name] = match;
 		}
+		// Otherwise if term is empty add all requested members
+		else if (term.is_empty()) {
+			if (search_flags & SEARCH_CONSTRUCTORS) {
+				for (int i = 0; i < class_doc.constructors.size(); i++) {
+					match.constructors.push_back(const_cast<DocData::MethodDoc *>(&class_doc.constructors[i]));
+				}
```

Comment:
I wonder if there would be a more concise way to create a `Vector<T *>` from a `Vector<T>` instead of doing it element by element. I don't have anything specific in mind, just mentioning it in case someone has an idea.